etc/docker-scripts/bootstrapper.py

```diff
@@ -176,12 +176,10 @@ def put_file_to_object_storage(client, bucket_name, file_to_upload, object_name=
     get_file_from_object_storage(cos_client, input_params['cos-bucket'], input_params['cos-dependencies-archive'])
 
     print('Processing dependencies........')
-    if 'inputs' in input_params.keys():
+    if input_params['inputs']:
         input_list = input_params['inputs'].split(",")
-        if input_list and len(input_list) > 0:
-            for file in input_list:
-                if file and file != 'None':
-                    get_file_from_object_storage(cos_client, input_params['cos-bucket'], file)
+        for file in input_list:
```

Comment:
similar case here if you have an empty 'inputs'?

Reply:
I have to agree with @lresende.  If the default of `input_params['inputs']` is `None` (when no option is provided), this will produce the following exception `AttributeError: 'NoneType' object has no attribute 'strip'`.

The empty string is also considered a Falsey, so you need to leave it as `if input_params['inputs']:` (and assume `parse_arguments()` adds all expected keys) or add the additional check:
```python
if 'inputs' in input_params and input_params['inputs']:
```

I personally prefer the latter so the reader/current maintainer doesn't have to question and/or wonder if parse_arguments adds entries even when not present (and optional).

Reply:
```python
foo['inputs'] = None
if 'inputs' in foo and foo['inputs'].strip():
    pass
Traceback (most recent call last):
  File "/opt/anaconda3/envs/elyra-dev/lib/python3.7/site-packages/IPython/core/interactiveshell.py", line 3319, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-2-a0f83a16910b>", line 1, in <module>
    if 'inputs' in foo and foo['inputs'].strip():
AttributeError: 'NoneType' object has no attribute 'strip'